src/main/java/dk/kb/util/Resolver.java

```diff
@@ -116,10 +118,10 @@ public static URL resolveURL(String resourceName)
      * @return a list of Paths matching the given glob or the empty list if there were no matches.
      */
     public static List<Path> resolveGlob(String glob) {
-        List<Path> paths = new ArrayList<>();
         if (glob == null || glob.isEmpty()) {
-            return paths;
+            return Collections.emptyList();
```

Comment:
Improvement to deliver an imutable empty list?

Reply:
Same behavior as before, I just found it to be cleaner code to delay the construction of a response structure until it has been checked whether there should be a "real" response or not. Also, explicitly using `Collections.emptyList();` is a very clear signal as to what is happening.